jest.eslint.config.js

```diff
@@ -2,6 +2,6 @@ module.exports = {
   runner: 'jest-runner-eslint',
   displayName: 'eslint',
   testMatch: ['<rootDir>/**/*.js'],
-  testPathIgnorePatterns: ['/node_modules/', '.public/', 'public/', 'dist/'],
+  testPathIgnorePatterns: ['/node_modules/', '.public/', 'public/', 'dist;'],
```

Comment:
is this on purpose?

Reply:
Nope. It's my rule of 👍 and life philosophy: no PR without a typo by Tobi. 3c869cb